java/core/src/main/java/com/redhat/rhn/domain/server/Server.java

```diff
@@ -2724,16 +2724,20 @@ boolean isDebian() {
         return ServerConstants.DEBIAN.equalsIgnoreCase(getOs());
     }
 
-    boolean isDebian12() {
-        return ServerConstants.DEBIAN.equals(getOs()) && getRelease().equals("12");
+    boolean isDebian10() {
```

Comment:
This strange diff is just me reordering as older to newer, as we do for all other OS.

For some odd reason this was newer to older until now.